### PR TITLE
Fixing data types

### PR DESCRIPTION
**Pull request by VincentLa**
In this PR I am

1. Fixing the Parsers to load data in with the correct data types
2. Adding Primary Keys to our tables
3. Refactoring the code for `trg_analytics.sba_zip_level` (just making the SQL tighter)
4. In `trg_analytics.sba_zip_level` only include zipcodes that are in our dataset. Thanks @makfan64 for loading this into our DB!

### CODE REVIEW

**Comment by VincentLa14** [comment]
I should also add `conda env create -f environment.yml` to docs

File: pipeline/pipeline_tasks/parse/00_03_01_load_irs_datasets.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def main():
     print('Parsing Census datasets')
     args = get_args()
     dbm = DBManager(db_url=args.db_url)
-    directory = 'C:/Users/User/Dropbox/Documents/Analytics/Analyses/SBA/data/'
+    directory = '/Users/VincentLa/git/datasci-sba/src/data/irs/'
```
Comment (VincentLa14) [inline pipeline/pipeline_tasks/parse/00_03_01_load_irs_datasets.py:40]: This will change soon. Figuring out Azure's data lake storage account.

**Review by gregboyer — APPROVED** [review]
Everything looks good to me. Thanks for doing some of the dirty work.

**Comment by VincentLa14** [comment]
Thanks! @fiascojazz

**Pull request merged**